Packages/vcs/Lib/Canvas.py

```diff
@@ -4718,10 +4718,7 @@ def ffmpeg(self, movie, files, bitrate=1024, rate=None, options=''):
             cmd += '-i %s_%%d.png' % (rnd)
         elif isinstance(files, str):
             cmd += '-i ' + files
-        if rate is not None:
-            cmd += ' -r %s ' % rate
-        if bitrate is not None:
-            cmd += ' -b:v %sk' % bitrate
+        cmd += " -pix_fmt yuv420p "
```

Comment:
@chaosphere2112 are `h264` encoded files now the new default for the `x.ffmpeg()` and the `x.animation()` functions (where `x = vcs.init()`)?